tests/suites/test_suite_aes.function

```diff
@@ -242,18 +253,20 @@ void aes_encrypt_cfb128( data_t * key_str, data_t * iv_str,
 
 
     mbedtls_aes_setkey_enc( &ctx, key_str->x, key_str->len * 8 );
-    TEST_ASSERT( mbedtls_aes_crypt_cfb128( &ctx, MBEDTLS_AES_ENCRYPT, 16, &iv_offset, iv_str->x, src_str->x, output ) == 0 );
+    TEST_ASSERT( mbedtls_aes_crypt_cfb128( &ctx, MBEDTLS_AES_ENCRYPT, 16,
+                 &iv_offset, iv_str->x, src_str->x, output ) == 0 );
 
-    TEST_ASSERT( hexcmp( output, hex_dst_string->x, 16, hex_dst_string->len ) == 0 );
+    TEST_ASSERT( memcmp( output, hex_dst_string->x, hex_dst_string->len )
+                 == 0 );
 
 exit:
     mbedtls_aes_free( &ctx );
 }
 /* END_CASE */
 
 /* BEGIN_CASE depends_on:MBEDTLS_CIPHER_MODE_CFB */
-void aes_decrypt_cfb128( data_t * key_str, data_t * iv_str,
-                         data_t * src_str, data_t * hex_dst_string )
+void aes_decrypt_cfb128( data_t *key_str, data_t * iv_str,
+                         data_t *src_str, data_t * hex_dst_string )
```

Comment:
Style: `data_t * iv_str` & `data_t * hex_dst_string` still have a whitespace after the `*`